draft-ietf-rtcweb-jsep.xml

```diff
@@ -2668,6 +2701,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   layers have been established. Otherwise, stop transmitting RTP media,
                   although RTCP should still be sent, as described in 
                   <xref target="RFC3264"/>, Section 5.1. </t>
+		<t>If the offer contained "a=rid" lines, then the "a=rid" lines,
```

Comment:
This is the wrong place to handle this. The section above parses the rid lines into their constituent values; here, we mention how to apply them.

(e.g. the text here should say something like "include the RID header extension in the RTP stream , as indicated in [doc], section foo)